QgisModelBaker/libqgsprojectgen/dbconnector/pg_connector.py

```diff
@@ -486,6 +486,41 @@ def get_min_max_info(self, table_name):
 
         return {}
 
+    def get_value_mapping_info(self, ili_name, field_name):
+
+        if not self._table_exists(PG_METAATTRS_TABLE):
+            return None
+
+        if not self.schema:
+            return None
+
+        cur = self.conn.cursor(cursor_factory=psycopg2.extras.DictCursor)
+        cur.execute(
+            """
+                    SELECT
+                      attr_value
+                    FROM {schema}.{metaattrs_table}
+                    WHERE LOWER(ilielement)=LOWER('{ili_name}.{field_name}')
```

Comment:
@signedav i think there should be a better variable than `field_name`, can you suggest me? (field_name is lowercase but in the model/db is not)

Reply:
I think `column_name` is fine.

Another approach would be to look up the ili-name of the attribute by the table `t_ili2db_attrname` or even return it by `get_fields_info` accessing this table already - you could do it analogue to `get_value_map` or am I overseeing something?

Reply:
Btw. the name `get_value_mapping_info` i find confusing. Wouldn't it be better like `get_list_info` or similar?

Reply:
thanks I removed completely `get_value_mapping_info` and return the mappings information with `get_fields_info`